apps/barnepensjon-ui/src/locales/en.ts

```diff
@@ -341,7 +341,8 @@ const aboutChildren = {
     onlyParentOrGuardianCanApply:
         'If the child has lost a parent, the parent or appointed guardian must submit a separate application.',
     onlyParentOrGuardianCanApplyOnLivingParent: `Only the child's parent or appointed guardian can apply for a children’s pension for this child. The children’s pension must be applied for separately.`,
-    onlyChildrenUnder18Necessary: 'You must only list children under the age of 18.',
+    onlyChildrenUnder18Necessary:
+        'You must only list children under the age of 18. Children over the age of 18 must apply themselves.',
```

Comment:
Mulig det bare er meg, men 'must apply themselves' oversetter jeg i hodet mitt til 'skjerpe seg' / 'legge inn instats'
Evt bruke: "apply for children's pension themselves'

Reply:
Går ut som den er nå. Justering av tekst tas i egen oppgave.